src/toxcore/tcp/codec.rs

```diff
@@ -101,6 +113,9 @@ impl Decoder for Codec {
     type Error = DecodeError;
 
     fn decode(&mut self, buf: &mut BytesMut) -> Result<Option<Self::Item>, Self::Error> {
+        // Add 1 to incoming counter
+        self.stats.write().incoming +=  1;
```

Comment:
Wait a minute, there should be another struct for tcp.

Like
```
struct Stats {
   tcp: {
       incoming,
       outgoing
   },
   udp {
       incoming,
       outgoing
   }
}
```

Reply:
https://github.com/tox-rs/tox/blob/59abfe1a11869ac40c293a802d8b5abfa4efbf2a/src/toxcore/tcp/codec.rs#L93

Reply:
the counter Stats objects are managed as separated object.

Reply:
First of all stats are dht stats because: `use toxcore::dht::codec::Stats;`